src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/ResourcesSpec.scala

```diff
@@ -0,0 +1,151 @@
+package ch.epfl.bluebrain.nexus.tests.kg
+
+import java.util.regex.Pattern.quote
+
+import akka.http.scaladsl.model.HttpMethods._
+import akka.http.scaladsl.model.{RequestEntity, StatusCodes, HttpRequest => Req}
+import ch.epfl.bluebrain.nexus.commons.http.JsonLdCirceSupport._
+import ch.epfl.bluebrain.nexus.tests.BaseSpec
+import org.scalatest.concurrent.Eventually
+import org.scalatest.{CancelAfterFailure, Inspectors}
+
+class ResourcesSpec extends BaseSpec with Eventually with Inspectors with CancelAfterFailure {
+
+  val orgId   = genId()
+  val projId1 = genId()
+  val id1     = s"$orgId/$projId1"
+
+  "cleaning ACLs" should {
```

Comment:
I would recommend mixing in the `BeforeAndAfterAll` trait and defining the clean-up as part of the `beforeAll` function instead.